server/controllers/get-ft-headlines.js

```diff
@@ -0,0 +1,33 @@
+const main = (req, res) => {
+
+	// process.env.FT_API = key
+	const fetch = require('isomorphic-fetch');
+	let options = {
+		method: 'POST',
+		body: {
```

Comment:
The body seems correct, but it needs to be a string 

`body: JSON.stringify({...})`

See https://github.com/github/fetch#post-json

Almost there!